views/site/books.php

```diff
@@ -7,174 +7,258 @@
-                </div>
-
-                <div class="col-sm-6 col-md-9">
-                    <h3>PHP Web Application Development <small class="authors">by Bill Keck</small></h3>
-
-                    <h4>Recommended for beginners.</h4>
-
-                    <p>It is a step by step introduction to the framework, which is based around
-                        creating a reusable template that can serve as the basis for your projects.</p>
+        </div> <!-- book -->
+        <div class="col-sm-3 col-md-3 col-lg-3">
+            <div class="[ info-card ]">
+                <div class="ribbon green"><span>Beginners</span></div>
+                <img src="<?= Yii::getAlias('@web/image/books/yii2-forbeginners.png') ?>" alt="PHP Web Application Development" />
+                <div class="[ info-card-details ] animate">
+                    <div class="[ info-card-header ]">
+                        <h1> PHP Web Application Development </h1>
+                        <h3> by Bill Keck </h3>
+                    </div>
+                    <div class="[ info-card-detail ]">
+                        <!-- Description -->
+                        <p>
+                            It is a step by step introduction to the framework, which is based around
+                                creating a reusable template that can serve as the basis for your projects.
+                        </p>
+                    </div>
+                    <div class="social">
+                    <a href="https://leanpub.com/yii2forbeginners">PHP Web Application Development</a>
+                    Recommended for beginners.
+                    </div>
                 </div>
             </div>
-
-            <div class="row book">
-                <div class="col-sm-6 col-md-3">
-                    <a href="http://www.amazon.com/dp/1783981881?tag=gii20f-20"><img src="<?= Yii::getAlias('@web/image/books/yii2-app-development.png') ?>" class="img-responsive" /></a>
-                </div>
-
-                <div class="col-sm-6 col-md-9">
-                    <h3>Web Application Development with Yii 2 and PHP <small class="authors">by Mark Safronov</small></h3>
-
-                    <h4>Recommended for intermediate-to-advanced developers.</h4>
-
-                    <p>The book is based around implementing a real world CRM application following many development best practices.
-                    It is assumed that reader is experienced enough in object orientented programming so don't expect OO-basics
-                    explained. There are many references to good overall programming books so even if you're already familiar with
-                    Yii 2.0 it worth at least skimming it.</p>
+        </div> <!-- book -->
+        <div class="col-sm-3 col-md-3 col-lg-3">
+            <div class="[ info-card ]">
```

Comment:
I will remove the brackets from the class names in a later commit - I probably should not add to this PR?